### PR TITLE
[JENKINS-35088] Migrate to new parent pom

### PR DESCRIPTION
**Pull request by andresrc**
See [JENKINS-35088](https://issues.jenkins-ci.org/browse/JENKINS-35088)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -46,22 +42,21 @@
   </scm>
 
   <properties>
-    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <project.build.outputEncoding>UTF-8</project.build.outputEncoding>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <jenkins.version>1.565</jenkins.version>
+    <java.level>6</java.level>
```
Comment (andresrc) [inline pom.xml]: Mapdb was Java 6, btw

File: pom.xml [hunk 2]
```diff
@@ -46,22 +42,21 @@
   </scm>
 
   <properties>
-    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <project.build.outputEncoding>UTF-8</project.build.outputEncoding>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <jenkins.version>1.565</jenkins.version>
```
Comment (andresrc) [inline pom.xml]: Small bump greatly simplifies getting along with newer hpi plugin and JTH, as well as PCT checks. It is still old enough (and even before 1.580.x) not be an issue imo.

**Comment by andresrc** [comment]
@reviewbybees

**Comment by stephenc** [comment]
:bee:

File: pom.xml (hunk 2 above)
Comment (stephenc) [inline pom.xml]: only an issue if current subversion plugin baseline is older than 1.565

Comment (andresrc) [inline pom.xml]: Ups, thanks for pointing out. Checked subversion is in 1.568

**Pull request merged**